.github/issue-labeler.yml

```diff
@@ -0,0 +1,11 @@
+# Add/remove 'bug' label if issue contains the words 'Bug' or 'bug'
+bug:
+  - '(Bug|bug)'
```

Comment:
Thinking again about this auto-labelling. We already set labels if user selects right template. For bug I omitted pushing directly bug label as it can be something else, so it is up to maintainers to add "bug" after checking...

Maybe, we can keep labels for PRs only without any issue labelers as I'm not sure about our usage for that...
What do you think ?

Reply:
@vfdev-5 Ok, now I see, I used bug template for testing and didn't notice that, for example, docs and question labels we add automatically on issue creating,

yes, I think in current configuration issue-labeler is useless for our situation, let's keep pr-labeler, and wait for more smarter issue-labeler :)